www/src/components/banner.js

```diff
@@ -17,7 +17,8 @@ function BannerContent() {
         href="https://www.gatsbyjs.com"
         sx={{
           color: `white`,
-          borderBottom: t => `1px solid ${t.colors.white}`,
+          borderBottom: 1,
+          borderColor: `white`,
```

Comment:
![image](https://user-images.githubusercontent.com/21834/86311652-2641a380-bc21-11ea-9751-4d0d15c9df25.png)

:(((

Reply:
Oh you know what, it's because `borders.js` isn't included in `theme-gatsbyjs-org.js`. Once it update that, it should work. Do you think you can do it?

Reply:
I was working on the review comments of tables PR, it is not working without using borderStyle: `solid` rule.

Reply:
Fixed here: https://github.com/gatsbyjs/gatsby/pull/25486 ;)

Reply:
🤦 **Thank you, Nat!** <3 <3 <3